snippets/card-product.liquid

```diff
@@ -338,28 +334,23 @@
   </div>
 {%- else -%}
   <div class="card-wrapper product-card-wrapper underline-links-hover">
-    <div
-      class="
-        card
-        card--{{ settings.card_style }}
-        card--text
+    <div class="card card--{{ settings.card_style }} card--media 
         {% if extend_height %} card--extend-height{% endif %}
         {% if settings.card_style == 'card' %} color-{{ settings.card_color_scheme }} gradient{% endif %}
-        {% if card_product.featured_media == nil and settings.card_style == 'card' %} ratio{% endif %}
-        {{ horizontal_class }}
-      "
+        {% if horizontal_class %} card--horizontal{% endif %}"
       style="--ratio-percent: 100%;"
     >
-      <div
-        class="card__inner {% if settings.card_style == 'standard' %}color-{{ settings.card_color_scheme }} gradient{% endif %}{% if settings.card_style == 'standard' %} ratio{% endif %}"
+      <div class="card__inner {% if settings.card_style == 'standard' %}color-{{ settings.card_color_scheme }} gradient{% endif %} ratio"
         style="--ratio-percent: 100%;"
       >
-        <div class="card__media">
-          {%- if placeholder_image -%}
-            {{ placeholder_image | placeholder_svg_tag: 'placeholder-svg' }}
-          {%- else -%}
-            {{ 'product-apparel-2' | placeholder_svg_tag: 'placeholder-svg' }}
-          {%- endif -%}
+        <div class="card__media color-{{ settings.card_color_scheme }} gradient"> 
```

Comment:
Curious to know if/when `color-{{ setings.card_color_scheme }}` and `gradient` are needed ? 🤔

Reply:
I believe we don't need it since we are referencing color-{{ setings.card_color_scheme }} and gradient in a parent div: "card__inner". I didn't realize I duplicated the class. Good catch :)